src/test/java/core/basesyntax/service/fruit/FruitServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service.fruit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.fruit.Operation;
+import core.basesyntax.model.fruit.Record;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceHandler;
+import core.basesyntax.strategy.impl.PurchaseHandler;
+import core.basesyntax.strategy.impl.ReturnHandler;
+import core.basesyntax.strategy.impl.SupplyHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class FruitServiceTest {
+    private static FruitService fruitService;
+    private List<Record> positiveRecords;
+    private List<Record> negativeRecords;
+    private List<Record> nullRecords;
```

Comment:
I think these fields are better to be removed and created directly in corresponding methods. Also, consider creating a separate test class for each handler